### ISSUE TITLE
React pdf Conflict with @swc/helpers@0.4.11

### ISSUE THREAD

**Issue by KhalidIrfani** [diegomura/react-pdf#2997]
Hi everyone!
I am getting this error the **@swc/helpers@0.4.11** with the react-pdf/render and i am using the next 13v
I do all the others way like cache clean , node module delete and reinstall and change the next version but didnt get any solution to resolve this issue
is there any way to resolved this issue

materio-mui-react-nextjs-admin-template@2.0.0 D:\alyzone\n-travel-scheduler\n_travel_web_dash
├─┬ **@react-pdf/renderer@3.4.5**
│ └─┬ @react-pdf/font@2.5.2
│   └─┬ fontkit@2.0.4
│     └── @swc/helpers@0.5.13
└─┬ next@13.0.0
  └── @swc/helpers@0.4.11

error - ./node_modules/fontkit/dist/browser-module.mjs:2:0
Module not found: Can't resolve '@swc/helpers/_/_define_property'

Import trace for requested module:
./node_modules/@react-pdf/font/lib/index.browser.js
./node_modules/@react-pdf/renderer/lib/react-pdf.browser.es.js
./src/pages/to/accounting/invoices/list/index.js

https://nextjs.org/docs/messages/module-not-found

**Comment by KhalidIrfani** [comment]
<img width="649" alt="save" src="https://github.com/user-attachments/assets/83fc3588-3833-4fe1-ae4b-53dd1bfcf724" />
this one is the issue which i getting

**Comment by KhalidIrfani** [comment]
> have you solve it?

Yes I resolved it by using the previous version of font kit and react pdf render
"@react-pdf/font": "^2.5.1",
 "@react-pdf/renderer": "^3.4.4",
 "fontkit": "^2.0.2",

**Comment by sabbircste** [comment]
@KhalidIrfani which next.js version are you using? i am using next.js 13.1.1 and failed to integrate even your suggested versions

**Comment by jdeniau** [comment]
After investigation the same issue : 

The issue comes because parcel, which bundle react-pdf, includes somes swc helpers.

We solve this by removing fontkit via the webpack configuration (in our case, react-pdf is not needed in nextjs). I do not know if this is viable in all case.

```ts
webpack(config, { dev, isServer }) {
    if (!isServer) {
      // eslint-disable-next-line no-param-reassign
      config.resolve.alias = {
        ...config.resolve.alias,
        fontkit: false,
      };
    }
```

**Comment by diegomura** [comment]
This is a dependency version conflict between Next.js 13 and fontkit's transitive dep on @swc/helpers. This was resolved in later versions. Please upgrade to Next.js 14+ or react-pdf v4. Closing as stale.
